SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,152 @@
+# Next steps: Set up the SpatialOS GDK Starter template
```

Comment:
Capitalisation is inconsistent within the doc - what is the proper noun?
Maybe`the GDK Starter template project` initially and call it the proper noun `GDK Starter template` thereafter and in all the other docs?? I kind of want "template" to be a proper noun too (we went for "tutorial" for the other project, I know). 
It's not a biggee but needs to be consistent

Reply:
Perhaps we should go with `SpatialOS GDK Starter Template' initially and then Starter Template thereafter?

Reply:
Sounds good.